eduNEXT/openedx-woocommerce-plugin
# fix: enrollment crud operations working correctly

## Description

The CRUD operation of the form was adjusted to work correctly in terms of data persistence, the ability to modify and delete records, and ensuring that the information updates accordingly. Moreover, any values that need to be changed (e.g., the title of the Enrollment Request) will be updated based on the fields that are modified. Additionally, unused code was removed to maintain clean code for the current tasks.

## Testing instructions

To test, simply access the plugin, create a new Enrollment Request, verify that it was saved correctly, and perform tests by modifying the existing variables to ensure that these changes persist.

## Additional information

The 'username' field has been removed, and only the 'email' field remains as the identifier.

## Checklist for Merge
